docs/src/howto/install/lxd.md

```diff
@@ -0,0 +1,196 @@
+# Install Canonical Kubernetes in LXD
+
+Canonical Kubernetes can also be installed inside an LXD container. This is a
+great way, for example, to test out clustered Canonical Kubernetes without the
+need for multiple physical hosts.
+
```

Comment:
Do we need prerequisits?

Reply:
I don't think so. The `multipass` install docs also do not have it. I'm not a big fan of putting something like "a good internet connection" as prerequisites.